share/wake/html/timeline.html

```diff
@@ -0,0 +1,795 @@
+            document.getElementById("directory").innerHTML = "";
+            document.getElementById("commandline").innerHTML = "";
+            document.getElementById("environment").innerHTML = "";
+            document.getElementById("stack").innerHTML = "";
+            document.getElementById("stdin_file").innerHTML = "";
+            document.getElementById("starttime").innerHTML = "";
+            document.getElementById("endtime").innerHTML = "";
+            document.getElementById("wake_start").innerHTML = "";
+            document.getElementById("wake_cmdline").innerHTML = "";
+            document.getElementById("stdout_payload").innerHTML = "";
+            document.getElementById("stderr_payload").innerHTML = "";
+            document.getElementById("usage").innerHTML = "";
+            document.getElementById("visible").innerHTML = "";
+            document.getElementById("inputs").innerHTML = "";
+            document.getElementById("outputs").innerHTML = "";
+            document.getElementById("tags").innerHTML = "";
+            return;
+        }
+        let job = jobMap.get(parseInt(properties.item)).job;
+        document.getElementById("job").innerHTML =job.job
+        if (job.usage.charAt(8) === '0') {
+            document.getElementById("job").style.color = "green";
+        } else {
+            document.getElementById("job").style.color = "red";
+        }
+        document.getElementById("stale").innerHTML =job.stale ? "true" : "false";
+        document.getElementById("label").innerHTML = job.label;
+        document.getElementById("directory").innerHTML = job.directory;
+        document.getElementById("commandline").innerHTML = job.commandline;
+        document.getElementById("environment").innerHTML = job.environment;
+        document.getElementById("stack").innerHTML = job.stack;
+        document.getElementById("stdin_file").innerHTML = job.stdin_file;
+
+        const starttime = new Date(job.starttime / 1e6);
+        document.getElementById("starttime").innerHTML = starttime.toISOString();
+        const endtime = new Date(job.endtime / 1e6);
+        document.getElementById("endtime").innerHTML = endtime.toISOString();
+        const wake_start = new Date(job.wake_start / 1e6);
+        document.getElementById("wake_start").innerHTML = wake_start.toISOString();
+
+        document.getElementById("wake_cmdline").innerHTML = job.wake_cmdline;
+        document.getElementById("stdout_payload").innerHTML = job.stdout_payload;
+        document.getElementById("stderr_payload").innerHTML = job.stderr_payload;
+
+        document.getElementById("usage").innerHTML = job.usage;
+        document.getElementById("visible").innerHTML = job.visible;
+        document.getElementById("inputs").innerHTML = job.inputs;
+        document.getElementById("outputs").innerHTML = job.outputs;
+        document.getElementById("tags").innerHTML = job.tags;
+    });
+
+    function compareJobs(jobA, jobB) {
```

Comment:
I would rename this to `jobsEqual()`. "Compare" makes it sound like it implements the comparator function interface (e.g. what you pass into `.sort()`), which returns a positive number when A > B, a negative number when B > A, and 0 when A == B.